variables.tf

```diff
@@ -40,6 +40,12 @@ variable "sql_identity_control_enabled" {
   default     = true
 }
 
+variable "azuread_authentication_only" {
+  description = "Is Azure Active Directory Authentication the only way to authenticate with resources inside this synapse Workspace?"
+  type        = bool
+  default     = false
```

Comment:
```suggestion
  default     = true
```
This should probably be set as true as this is more close to the baseline in my opinion.

Reply:
Good point. I agree